src/launch/config.c

```diff
@@ -380,6 +380,8 @@ static int config_parser_attrs_allow_deny(ConfigState *state, ConfigNode *node,
         char *t;
         int r;
 
+        node->allow_deny.max_fds = UINT64_MAX;
+
```

Comment:
meh... That doesn't belong here... Can't we put that in config_node_new() just like config_node_free() does the cleanup?

Reply:
Agreed. Fixed.